presto-plugin-toolkit/src/test/resources/file-based-system-access-schema.json

```diff
@@ -0,0 +1,33 @@
+{
```

Comment:
I think we need a better way to injecting rules to tests. It would be nice to define them next to the test as java object, then do serialization round trip, and the do the test. Maybe this should go as a refactor before this commit or after.

Then instead of giant test with many cases we could have plenty of smaller test per case.

Reply:
@kokosing - I get your point. Think we can do it after this commit. Will create a followup ticket to track it.